vagrant/playbooks/source-install.yml

```diff
@@ -12,11 +12,34 @@
 
     # The version string below is the highest of all those in roles' metadata:
     # "min_ansible_version". It needs to be kept manually up-to-date.
+    # FIPS boxes require 2.10
     - name: Verify Ansible meets min required version
       assert:
-        that: "ansible_version.full is version_compare('2.8', '>=')"
+        that: "ansible_version.full is version_compare('2.10', '>=')"
         msg: >
-          "You must update Ansible to at least 2.8 to use this version of Pulp 3 Installer."
+          "You must update Ansible to at least 2.10 to use this version of Pulp 3 Installer."
+
+    - name: Building pulp.pulp_installer
+      command:
+        cmd: ansible-galaxy collection build --force .
+        chdir: ../..
+      delegate_to: localhost
+
+    - name: Installing pulp.pulp_installer
+      command:
+        cmd: "ansible-galaxy collection install --force {{ item }}"
+        chdir: ../..
+      with_fileglob:
+        - "../../pulp-pulp_installer-*.tar.gz"
+      delegate_to: localhost
+
+    - name: Clean built collection
+      file:
+        state: absent
+        path: "{{ item }}"
+      with_fileglob:
+        - "../../pulp-pulp_installer-*.tar.gz"
+      delegate_to: localhost
```

Comment:
Maybe it's better to swap the order here. First clean out the old files, then build the fresh collection, then install the single fresh one.